qase-python-commons/src/qase/commons/profilers/network.py

```diff
@@ -68,15 +70,25 @@ def _log_pre_request(request):
             data=StepRequestData(
                 request_method=request.method,
                 request_url=request.url,
-                request_body=request.body,
-                request_headers=request.headers,
+                request_body=request.body if hasattr(request, 'body') else None,
+                request_headers=request.headers if hasattr(request, 'headers') else None
             ),
         )
 
     @staticmethod
     def _log_post_response(response, url=None, *args, **kwargs):
```

Comment:
post means the HTTP POST method or "post response" as "after response"?

Reply:
it is like "after response". I will rename the method

Reply:
fixed